tools/misc/bad_mapping_remover.py

```
@@ -0,0 +1,126 @@
+    header = [cell.value for cell in sheet[1]]
+    if "node_id" not in header:
+        return set(), False
+    idx = header.index("node_id")
+    ids = set()
+    for row in sheet.iter_rows(min_row=2):
+        if idx < len(row) and row[idx].value:
+            ids.add(str(row[idx].value).strip())
+    return ids, True
+
+
+def main(file_path, mapping_sheet_name="mappings_content"):
```

Pylint (3.3.7)

[refactor] Too many local variables (32/15)

(R0914)